data/json/items/comestibles/other.json

```diff
@@ -1086,6 +1086,22 @@
     "freezing_point": -274,
     "flags": [ "NUTRIENT_OVERRIDE" ]
   },
+  {
+    "id": "swarm_bees",
+    "type": "COMESTIBLE",
+    "comestible_type": "FOOD",
+    "name": { "str": "swarm of bees", "str_pl": "swarms of bees" },
+    "description": "A swarm of honey bees, displaced from the hive.  Without support and nutrients, or taking root somewhere, they will die in a few days.",
+    "//": "While bees have mass and volume, it is easier to round to zero for the purposes of the game.",
+    "price": 0,
+    "price_postapoc": 0,
+    "spoils_in": "2 days",
+    "flags": [ "ZERO_WEIGHT", "INEDIBLE" ],
+    "volume": "500 ml",
```

Comment:
Spilled beeswarms being lost notwithstanding I think it really should work as a liquid for our purposes unless liquid forage results derp out.

Reply:
Forage results derp out when this is set as a liquid. It instantly drops to the ground, becomes "dirty", and leaves the player with no way to pick it up.

Reply:
Okay, weirdly complacent swarms it is I suppose. Might be worth a feature request for future reference.